setup.py

```diff
@@ -1068,6 +1068,23 @@ def make_relative_rpath(path):
         ]
     }
 
+
+def collect_data_files(directory):
```

Comment:
Since we modify the CMake file, is this part of the change is needed?

Reply:
I think it would be necessary if this were to be run from setup.py. It's here that we choose between cmake and setup.py. Even though it's not currently an "integrated" test, I guess it could be in the future. I can take this out or leave it in, your choice.
https://github.com/pytorch/pytorch/blob/master/.jenkins/caffe2/build.sh#L199

Reply:
The question is that if we install the caffe2 using `FULL_CAFFE2=1 python setup.py install`, is the change on CMakeLists is already enough or we still need the changes in setup.py. Could you verify this and make sure we don't bring in any unnecessary change?

Reply:
I manually verified that if we do not modify setup.py, that the expected binary files don't show up in the output path used by setuptools, `/home/ansha/venv-install/lib64/python2.7/site-packages/caffe2/python/serialized_test/data`

Reply:
ok, then let's keep this change.